api/server/routes/aas.py

```diff
@@ -26,6 +27,7 @@ async def create_aas(request: Request) -> Any:
 
         @self.router.get("/shells/$reference")
         async def get_all_aas_reference() -> Any:
+            # TODO: Inconsistent
             return {"message": "Content parameters are not supported yet."}
```

Comment:
```suggestion
            # TODO: We should make handling of not implemented features consistent across the whole server.
            return {"message": "Content parameters are not supported yet."}
```

We might forget in a few weeks what we meant here 😅

Reply:
As far as I can see this is also the single, unimplemented route that is not consistent with the others. So I replaced this with the usual
`raise HTTPException(status_code=501, detail="This route is not yet implemented!")`